src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -75,6 +117,7 @@ export class TypeOfCarePage extends React.Component {
       data,
       pageChangeInProgress,
       showToCUnavailableModal,
+      address,
```

Comment:
It doesn't look like you use this. You could change it to `addressLine1`, then below on line 131 you can do `addressLine1` instead of `this.props.addressLine1`.